container-storage-setup.sh

```diff
@@ -1336,6 +1376,8 @@ setup_storage_compat() {
   if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
     setup_lvm_thin_pool_compat
   else
+      # Handle the try-overlay case here, otherwise just write the config
+      consume_try_overlay
```

Comment:
I think this call should come before the devicemapper check? First we need to select a storage driver and then get into the code which sets it up.

Reply:
I did that in the `fixup!` - try reviewing via the full "Files changed" tab.